README.md

```diff
@@ -129,16 +133,16 @@ sudo apt-get install mono-complete
 
 Now, verify your version of Mono by running ```mono --version```. Version 4.6 and above should be good enough.
 
-## Running .NET Core 2.2.6
+## Running .NET Core 3.0
 
-This project can also run in .NET Core. To install .Net Core 2.2.6 runtime please execute the following commands:
+This project can also run in .NET Core. To install .Net Core 3.0 sdk please execute the following commands:
 
 ```
 $ sudo apt-get -y update
 $ sudo apt-get -y install libunwind8 gettext
-$ wget https://download.visualstudio.microsoft.com/download/pr/428aaa32-f66c-4847-b845-aa21f90504e4/1cf033db866414997140c2672bd75069/dotnet-runtime-2.2.6-linux-arm.tar.gz
+$ wget https://download.visualstudio.microsoft.com/download/pr/8ddb8193-f88c-4c4b-82a3-39fcced27e91/b8e0b9bf4cf77dff09ff86cc1a73960b/dotnet-sdk-3.0.100-linux-arm.tar.gz
 $ sudo mkdir /opt/dotnet
-$ sudo tar -xvf dotnet-runtime-2.2.6-linux-arm.tar.gz -C /opt/dotnet/
+$ sudo tar -xvf dotnet-sdk-3.0.100-linux-arm.tar.gz -C /opt/dotnet/
```

Comment:
Any particular reason to recommend the SDK over the runtime?

Reply:
Not really, actually. So far I didn't need the SDK on the pi itself at all. But on the other hand, I haven't checked for the differences and it doesn't hurt ;-) Maybe we should link both.

Reply:
The runtime includes only what you need to run .NET Core applications. The SDK includes everything you need to build and run .NET Core applications, which means that the SDK also includes the Runtime.
Since we only want to show how to run the APPs on the Raspberry PI, it is not necessary to install the full SDK.